src/config_generator/from_proto.rs

```diff
@@ -330,6 +336,8 @@ fn convert_ty(proto_ty: &str) -> String {
         "int32" | "int64" | "fixed32" | "fixed64" | "uint32" | "uint64" => "Int",
         "bool" => "Boolean",
         "string" | "bytes" => "String",
+        "message" => "JSON", /* JSON scalar is preloaded by tailcall, so there is no need to
+                               * explicitly define it in the config. */
```

Comment:
Use `//` to write comments in code. We do that everywhere else.

Reply:
it was converted to /* by cargo or prettier